src/MagnumPlugins/GltfImporter/Test/mesh-unsupported-vertex-formats.gltf

```diff
@@ -36,6 +36,22 @@
           "mode": 0
         }
       ]
+    },
+    {
+      "primitives": [
+        {
+          "attributes": {
+            "POSITION": 1
+          },
+          "targets": [
+            {
+              "WEIGHTS": 4,
```

Comment:
It took me a considerable amount of time to realize why the CI nondeprecated build (i.e., a build with `MAGNUM_BUILD_DEPRECATED` disabled) throws :sweat_smile: It's because the default build, with deprecated features enabled, has custom named `WEIGHTS` and `JOINTS` attributes for backwards compatibility, which were used before there was builtin skinning support. And if the deprecated features are turned off, the `WEIGHTS` and `JOINTS` are not in `_d->meshAttributesForName`, causing an exception to be thrown in `at()`.

You'd fix the consequence by changing this to the actual correct attribute name:

```suggestion
              "WEIGHTS_0": 4,
```

But the root cause is that custom attribute names in morph targets are not collected. I.e., the following (although an invalid glTF, because the morph target should reference only attributes that are actually in the mesh) would blow up too:

```json
    {
      "primitives": [
        {
          "attributes": {
            "POSITION": 1
          },
          "targets": [
            {
              "_FANCY_PROPERTY3": 4
            }
          ]
```

To fix this, the ["Go through all meshes, collect custom attributes" block](https://github.com/mosra/magnum-plugins/blob/90a0da68f17fbe7758271d64ed02c4a822a23755/src/MagnumPlugins/GltfImporter/GltfImporter.cpp#L1241-L1327) in `doOpenData()` needs to go through `"targets"` subobjects as well, not just `"attributes"`, and do basically the same. 

I think the change should be manageable by extracting the inner block that loops over the attributes into a lambda and calling it for both `"targets"` subobjects and `"attributes"`. This in turn means you move most of (all?) the new `gltf->parseWhatever()` calls and error checking directly into `doOpenData()` and in `doMesh()` you'll access the already-parsed tokens directly with `->asWhatever()`, relying on them being already checked to be correct. 

Three out of the four new failure tests thus move from `MeshInvalidData[]` into `MeshInvalidWholeFileData[]` as they now cause an error during file opening already, and from `mesh-invalid.gltf` into three new files (which means one more painful line renumbering).

For actually verifying that the custom attribute names get imported (so, that in the above snippet the `_FANCY_PROPERTY3` gets recognized and available in `importer->meshAttributeName()` and such) you'd expand the `mesh-custom-attributes.gltf` file and the `meshCustomAttributes()` case. That should be everything needed I think.